src/clj_icat_direct/queries.clj

```diff
@@ -194,17 +195,17 @@
   [dirname basename group-ids-query]
   [(str "WITH "
      "objs AS ("
-       "SELECT c.coll_id AS object_id, 'collection' AS type, c.coll_name AS full_path, regexp_replace(c.coll_name, '.*/', '') AS base_name, 0 as data_size, c.create_ts, c.modify_ts FROM r_coll_main c WHERE coll_name = ? || '/' || ? "
+       "SELECT c.coll_id AS object_id, 'collection' AS type, c.coll_name AS full_path, regexp_replace(c.coll_name, '.*/', '') AS base_name, 0 as data_size, c.create_ts, c.modify_ts, NULL AS data_checksum FROM r_coll_main c WHERE coll_name = ? || '/' || ? "
```

Comment:
(applies several spots, choosing one at random)

wasn't sure, and you may have tested this already, but I'm wondering if adding this column will break schema validation for any listings. We might need to either filter it down to the supported keys (but for all I remember we might already) or add the checksum to the listings (which might be the better option, but more out of scope)

Reply:
I spot checked a few places, and I think that we're filtering the list of fields down in most places. Admittedly, I haven't done an exhaustive search. I'll make sure to hit all of the endpoints before merging the changes. If I do have to make changes, I'll probably put them in data-info or terrain.

Reply:
Sounds good to me. I think data-info (or maybe terrain) would definitely be the right place to do it, glad to have it added in here.